test/test_mps.py

```diff
@@ -1467,17 +1467,19 @@ def _linear_helper(self, in_features, out_features, shape, bias=True, backward_p
         self.assertEqual(linear_cpu_output.size(), linear_mps_output.size())
 
         if backward_pass:
-            cpu_grad = torch.ones_like(linear_cpu_output)
+            cpu_grad = torch.rand_like(linear_cpu_output)
             grad = cpu_grad.to('mps')
 
             linear_cpu_output.backward(gradient=cpu_grad)
             linear_mps_output.backward(gradient=grad)
 
             self.assertEqual(linear_cpu_input.grad.size(), linear_mps_input.grad.size())
             self.assertEqual(linear_cpu_input.grad, linear_mps_input.grad.to("cpu"), atol=8e-04, rtol=10.4e-05)
+            self.assertTrue(gradgradcheck(mps_linear, linear_mps_input, grad))
 
             self.assertEqual(cpu_linear.weight.grad.size(), mps_linear.weight.grad.size())
             self.assertEqual(cpu_linear.weight.grad, mps_linear.weight.grad.to("cpu"), atol=8e-04, rtol=10.4e-05)
+            self.assertTrue(gradgradcheck(mps_linear, mps_linear.weight, atol=7e-03, rtol=10.4e-05))
```

Comment:
Can we also check gradgradcheck on bias?
Also to simplify things we can call to gradgradcheck with all the inputs passed in at the same time: input, weight, bias. It would also check more combinations of undefined grad handling.

Reply:
Yes indeed, the bias term should be considered. I will update that!